server/cmd/mmctl/commands/team.go

```diff
@@ -191,15 +192,20 @@ func archiveTeamsCmdF(c client.Client, cmd *cobra.Command, args []string) error
 	for i, team := range teams {
 		if team == nil {
 			printer.PrintError("Unable to find team '" + args[i] + "'")
+			errorString = *multierror.Append(&errorString, errors.New("Unable to find team '"+args[i]+"'"))
 			continue
 		}
 		if _, err := c.SoftDeleteTeam(context.TODO(), team.Id); err != nil {
 			printer.PrintError("Unable to archive team '" + team.Name + "' error: " + err.Error())
+			errorString = *multierror.Append(&errorString, errors.New("Unable to archive team '"+team.Name+"' error: "+err.Error()))
 		} else {
 			printer.PrintT("Archived team '{{.Name}}'", team)
 		}
 	}
 
+	if errorString.Len() > 0 {
+		return &errorString
+	}
 	return nil
```

Comment:
`ErrorOrNil` makes this easier to read.

```suggestion
	return result.ErrorOrNil()
```